pkg/users/users.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package users
+
+import (
+	"strings"
+
+	galasaErrors "github.com/galasa-dev/cli/pkg/errors"
+)
+
+func validateLoginIdFlag(loginId string) error {
+
+	var err error
+
+	if loginId == "" || strings.TrimSpace(loginId) == "" {
+		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_MISSING_USER_LOGIN_ID_FLAG)
+	}
+
+	if err == nil {
```

Comment:
Can we trim the string and use the trimmed content ? currently we trim it to check for blank, but throw it away.

Reply:
Probably have to return the modified loginId so the caller can use it.